### ISSUE TITLE
Simple Guide fro Voron Trident

### ISSUE THREAD

**Issue by pasha4ur** [jlas1/Klicky-Probe#218]
Hello

Can you PLEASE write simple guide for Voron trident using KISS principle? https://en.wikipedia.org/wiki/KISS_principle

Because at this moment all info was written with principle keep it F*CKING hard, IMHO. (

You divided all in separate files and now few existed manuals on youtube are irrelevant.

I built my 1 selfmade printer with a lot mods and only your mod is so difficult for me due to unstructured incomplete and scattered information.

This is not rocket science. Please write instructions in a simpler way.

**Comment by rogerlz** [comment]
yeah @pasha4ur, it is not rocket science and thousands of people, including myself, are using this project with the provided documentation.
If you have a better way, please open a pull request.

Thanks JosAr (@jlas1) for the hard work 👍
